### PR TITLE
renamed Back button in games

### LINKED ISSUE #40: Renaming back button in games
I find it hard to distinguish between Back and Undo during games.
Wouldn't it be better if the Back button here would be called Exit instead?
And Kilépés in Hungarian.

### PR DESCRIPTION
**Pull request by Btimes2**
This commit and this branch closes #40.

I also reordered the yes / no and on/off labels in the Hungarian translation so that the keys match exactly the English ones, but those change are not important and you may just ignore them if they would cause any conflicts later.

### CODE REVIEW

File: asset/translations/EN.lua [hunk 1]
```diff
@@ -13,6 +13,7 @@ return {
     gameover = "Game over",
     backtomainmenu = "Back to main menu",
     back = "Back",
+    back_ingame = "Exit",
```
Comment (Sasszem) [inline asset/translations/EN.lua]: This line is not needed

File: asset/translations/HU.lua [hunk 2]
```diff
@@ -13,15 +13,16 @@ return {
     gameover = "Játék vége",
     backtomainmenu = "Vissza a menübe",
     back = "Vissza",
+    back_ingame = "Kilépés",
```
Comment (Sasszem) [inline asset/translations/HU.lua]: This line is not needed.

File: src/menu/GameWrapper.lua [hunk 3]
```diff
@@ -14,7 +14,7 @@ local BS = require("src.menu.buttonStyle")
 local GameWrapper = VDiv(
     Label("", {span=11}),
     HDiv(
-        Button(L["back"], BS, "backFromGame"),
+        Button(L["back_ingame"], BS, "backFromGame"),
```
Comment (Sasszem) [inline src/menu/GameWrapper.lua]:
```suggestion
        Button(L["exit"], BS, "backFromGame"),
```

**Review by Sasszem — DISMISSED** [review]
We already have an `exit` key in translations.
Also, please remove trailing newlines!

**Comment by Btimes2** [comment]
Sorry, I was so proud of myself to be able to use git that I did not check the existing translations :)

**Comment by Btimes2** [comment]
I made the requested changes, how do I proceed?

**Comment by Btimes2** [comment]
Oh hey, what is a trailing newline?

**Comment by Sasszem** [comment]
> I made the requested changes, how do I proceed?

Dismiss my review, request a new

> Oh hey, what is a trailing newline?
Your IDE adds a `\n` after the last line. That's optional and in this project I decided not to have it. I suggest trying to re-save the file (maybe after deleting the empty line) via `nano` for example.

**Review by Sasszem — APPROVED** [review]
Removed trailing newlines, code is perfect.

**Pull request merged**